AutomationRebuild/src/test/java/framework/DriverManagerFactory.java

```diff
@@ -0,0 +1,17 @@
+package framework;
+
+public class DriverManagerFactory {
+
+	public static DriverManager getManager(String browser) {
+		DriverManager manager;
+		
+		if(browser.equalsIgnoreCase("chrome")) {
+			manager = new ChromeDriverManager();
+		}
+		else {
+			manager = new EdgeDriverManager();
+		}
+		return manager;

```

Comment:
This code defaults to Edge. So if an invalid browsertype was specified in the config file then edge will open. Throw an exception instead, if unsupported browsertypes are passed.

Reply:
Code now throws FileNotFound exception when unsupported browsertype is passed.